requirements_dev.txt

```diff
@@ -0,0 +1,7 @@
+# Useful development packages.
+# Install them with 'pip install -r requirements_dev.txt'.
+
+memory-profiler
+pudb
+pyinstrument
+vmprof; platform_machine == "x86_64"
```

Comment:
FWIW, `pip install vmprof` fails for me, I have to install their repo. Maybe mention that?

Reply:
Weird, it works for me. I'll add a note.